manager/src/main/java/io/gingersnapproject/database/vendor/VendorUtil.java

```diff
@@ -0,0 +1,25 @@
+package io.gingersnapproject.database.vendor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.gingersnapproject.database.model.PrimaryKey;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+
+/**
+ * TODO!
+ */
+public enum VendorUtil {
```

Comment:
Why an `enum`?

Reply:
No particular reason. It is just an alternative way of creating a private class with a private constructor to prevent any instantiation.

Reply:
moved the method to `Vendor` interface.